### PR TITLE
Remove obsolete script upgrade_send_spool

### PR DESCRIPTION
**Pull request by racke**
Direct update of 6.1 will no longer be supported, so this script isn't needed anymore. See #1091.
Not tested yet.

### CODE REVIEW

**Comment by dverdin** [comment]
Hi,
That simplifies certainly the code but how will people upgrade from Sympa prior to 6.2?

**Comment by racke** [comment]
They need to use an intermediary version. I don't see a point in upgrading from a version that is more than five years old.

**Comment by dverdin** [comment]
Sympa handles upgrade from the oldest version possible, i you look at Upgrade.pm. Removing upgrade_send_spool.pl will break this functionality.
I don't understand how to use an intermediate version. They will still need to upgrade the send spool.
Do you remove this script only because of the cookie parameter? Or is it hard to maintain as a packager?

**Comment by racke** [comment]
I really would like to move on with Sympa and not keep archaic code forever. Users upgrading from 6.1 can go first to any version in 6.2. It is ridiculous to carry around that code for a few users which may have affected and sitting on a stale and insecure Sympa version.

**Comment by dverdin** [comment]
I certainly understand the argument about archaic code.
However, if they have old versions, it means they are long-term users.
I don't think it is fair to leave them without any solution to move to the newest version.
The script should at least move to contributions, with notes about how to use it.

**Comment by racke** [comment]
The solution is to upgrade to 6.2.60 first and then to 6.2.xx. How can that be a problem?

**Comment by dverdin** [comment]
OK, got it. Correctly documented, people could still upgrade, even from very old versions.
I still think that the question of version upgrade deserves an issue of its own.

**Comment by ikedas** [comment]
> The solution is to upgrade to 6.2.60 first and then to 6.2.xx. How can that be a problem?

I once thought of putting such "check point", too.  However, I rethought:
  - Codes for measure on incompatibility are limited in `Sympa::Upgrade` and support tools under `src/bin`.
  - `Sympa::Upgrade` will be updated incrementally.  This means that gone versions won't affect to changes in the future.
  - Why 6.2.x lasts for long time is that quality of code is too poor to change focus of development to the new major release.  There are no reason to add a discontinuity at midst of 6.2.x.

In short, I think we  should dedicate our time to remove harmful things, keeping harmless things untouch.

**Comment by racke** [comment]
These things are not harmless, they are wasting time when navigating or search the source code.

**Comment by racke** [comment]
And I really can't fathom a new major release at this point.

**Comment by ikedas** [comment]
> These things are not harmless, they are wasting time when navigating or search the source code.


> And I really can't fathom a new major release at this point.

Honestly, I also can't imagine new major release as an extension of the current work  (At least some components including wwsympa would be faster to rewrite, if we were willing).

Besides, how about submitting new issue to consider what should be removed from recent 6.2? There may also be the others than codes for upgrade.

**Comment by racke** [comment]
Apparently @ikedas and @dverdin want to keep this script against reasonable arguments.
